prometheus-to-sd/config/gce_config.go

```diff
@@ -17,10 +17,23 @@ limitations under the License.
 package config
 
 import (
+	"flag"
 	"fmt"
 	"strings"
 
 	gce "cloud.google.com/go/compute/metadata"
+	"github.com/golang/glog"
+)
+
+var (
+	// ManualProject is a flag that allows a different project to be set when the
+	// GCE Metadata server is not available.
+	ManualProject = flag.String("manual-project", "",
+		"Custom GCP project to send metrics to. Can be used when running outside GCE.")
+	manualCluster = flag.String("manual-cluster", "",
+		"Custom cluster name to use. Useful when running outside GCE.")
+	manualInstance = flag.String("manual-instance", "",
```

Comment:
Can you name the flag "node_name"? "instance" is GCE-specific term, while "node" is k8s term.

The old gke_container monitored resource requires GCE instance_id. prom-to-sd ignores this convention and uses instance_name instead. New monitored resources k8s_container, k8s_pod, k8s_node (which should be recommended) require node name.